ExampleAppUITests/TradeItIosTicketSDK2UITests.swift

```diff
@@ -172,67 +397,223 @@ class TradeItIosTicketSdk2UITests: XCTestCase {
+        }
+        else if (longBrokerName == "dummySecurity"){
+            usernameTextField.typeText("dummySecurity")
+            passwordTextField.tap()
+            waitForElementToHaveKeyboardFocus(passwordTextField)
+            waitForElementNotToHaveKeyboardFocus(usernameTextField)
+            passwordTextField.typeText("dummy")
+        }
+        else {
+            usernameTextField.typeText("dummy")
+            passwordTextField.tap()
+            waitForElementToHaveKeyboardFocus(passwordTextField)
+            passwordTextField.typeText("dummy")
+        }
+        app.buttons["Link Broker"].tap()
         
         let activityIndicator = app.activityIndicators.element
         waitForElementNotToBeHittable(activityIndicator, withinSeconds: 10)
     }
     
-    private func selectFirstAccountOnthePortfolioScreen(app: XCUIApplication) {
+    private func selectAccountOnthePortfolioScreen(app: XCUIApplication, rowNum: Int) {
         waitForElementToAppear(app.navigationBars["Portfolio"])
-
+        
         var ezLoadingActivity = app.staticTexts["Authenticating"]
         waitForElementToDisappear(ezLoadingActivity, withinSeconds: 10)
-
+        
         ezLoadingActivity = app.staticTexts["Retreiving Account Summary"]
         waitForElementToDisappear(ezLoadingActivity)
         
         let tableView = app.tables.elementBoundByIndex(0)
         XCTAssertTrue(tableView.cells.count > 0)
         
-        let firstCell = tableView.cells.elementBoundByIndex(1) // index 0 is header
-        XCTAssertTrue(firstCell.images["selectorLabel"].exists)
+        let cell = tableView.cells.elementBoundByIndex(UInt(rowNum)) // index 0 is header
+        cell.tap()
+        
     }
-
-//    func testPortfolioUserHasAccountFlow() {
-//        // Launch ticket
-//        let app = self.application
-//        
-//        // See Portfolio screen
-//        app.tables.staticTexts["LaunchPortfolio"].tap()
-//        XCTAssert(app.navigationBars["Portfolio"].exists)
-//        
-//        var ezLoadingActivity = app.staticTexts["Authenticating"]
-//        waitForAsyncElementToDisappear(ezLoadingActivity)
-//        
-//        ezLoadingActivity = app.staticTexts["Retreiving Account Summary"]
-//        waitForAsyncElementToDisappear(ezLoadingActivity)
-//        
-//        XCTAssert(app.tables.cells.count > 0)
-//        
-//        //Balances
-//        app.tables.staticTexts["Dummy *cct1"].exists
-//        app.tables.staticTexts["$2408.12"].exists
-//        app.tables.staticTexts["$76489.23 (22.84%)"].exists
-//        
-//        //Positions
-//        app.tables.staticTexts["Dummy *cct1 Holdings"].exists
-//        app.tables.staticTexts["AAPL (1)"].exists
-//        app.tables.staticTexts["$103.34"].exists
-//        app.tables.staticTexts["$112.34"].exists
-//    }
+    
+    private func testPortfolioValues(app: XCUIApplication, brokerName: String){
+        
+        if(brokerName == "Dummy"){
```

Comment:
You're using `brokerName` a lot -- @mitochondrion what's the correct wording here? Broker name makes me think of the name of the broker.